app/controllers/import_controller.rb

```diff
@@ -278,6 +279,17 @@ def import_date_field(value)
     value.blank? ? nil : value
   end
 
+  def import_time_field(field, value, row_ind)
+    return nil if value.blank?
+
+    normalized_value = value.to_s.downcase.strip.gsub(/[ -.]/, '')
+    saved_value = NORMALIZED_INVERTED_TIME_OPTIONS[normalized_value]
+    return saved_value unless saved_value.blank?
+
+    err_msg = "Value '#{value}' for '#{VALIDATION[field][:label]}' is not an acceptable value, acceptable values are: '#{TIME_OPTIONS.values.join("', '")}'"
+    raise ValidationError.new(err_msg, row_ind)
```

Comment:
I think for maximum consistency, this function should be called `import_and_validate_time_field`, since this validation is happening here.

Reply:
Actually, could we instead just fall back on the model validation here? It seems like we could do this validation on the model, although we may need to define it separate from other `enum` validation so that the `TIME_OPTIONS` are displayed correctly.

Reply:
that was my initial approach, but then the validation message would say something like '...acceptable values are: [0, 1, 2, 3, 4...' instead of '...acceptable values are: [Midnight, 1:00, 2:00, 3:00...'

Reply:
I still renamed it though for consistency: https://github.com/SaraAlert/SaraAlert/pull/1019/commits/daa0195b5441006dc8259d33d13da45f325d66c6#diff-ee31e076a80cc9f48da0394c0cb066d59bf2f8c68f68eb62cb5119d01712d90eR246